.github/workflows/docker-publish.yml

```diff
@@ -9,9 +9,17 @@ on:
 jobs:
   docker:
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: write
+      security-events: write
+
     steps:
-      - name: Checkout
-        uses: actions/checkout@v4
```

Comment:
we dont need checkout code?

Reply:
not for this method of using buildx,

Reply:
I don't understand what you mean. If we don't checkout the source code, can we do docker build? Have you tested it in the github action environment? 🤔

Reply:
Tested, docker/build-push-action will pull source code by it self